### PR TITLE
vscode-14 fix component view on osx/linux

### LINKED ISSUE #14: Component preview is empty on unix
Bug only on OS X and linux
The component preview displays a white panel instead of the diagram.

### PR DESCRIPTION
**Pull request by nicolaserny**
it was a problem with the bundle path in the script tag
fixes #14 

@mbarekh could you test on windows too ?

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
 	"showOutput": "silent",
 
 	// we run the custom script "compile" as defined in package.json
```
Comment (maurelio1234) [inline .vscode/tasks.json]: remove this comment?

Comment (maurelio1234) [inline .vscode/tasks.json]: all of them? why do we need them?

Comment (nicolaserny) [inline .vscode/tasks.json]: it's a generated file from vscode (containing settings to start an extension)
I've just modified a parameter to fix the compile task

Comment (nicolaserny) [inline .vscode/tasks.json]: Done comment updated

**Pull request merged**